test/uk/gov/hmrc/leakdetection/services/ScanningServiceSpec.scala

```diff
@@ -18,8 +18,7 @@ package uk.gov.hmrc.leakdetection.services
 
 import java.io.{File, PrintWriter}
 import java.nio.file.Files
-import java.time.{Instant, Duration}
-
+import java.time.{Duration, Instant}
 import ammonite.ops.Path
 import com.typesafe.config.ConfigFactory
 import org.mockito.ArgumentMatchers.{any, eq => is}
```

Comment:
`is` should be `eqTo` for consistency (already provided by `ArgumentMatchersSugar`)